scst-store/configuration/use-load-balancer.hbs.md

```diff
@@ -0,0 +1,39 @@
+# Using LoadBalancer
+
+To use a `LoadBalancer` configuration, you must find the external IP address of the `metadata-store-app` service by using kubectl.
```

Comment:
As part of the introduction, you should mention that the recommended service type is ingress and link to the new `ingress.hbs.md`.

Reply:
Added, does the wording look okay to you?

Reply:
Yes that looks good thank you